src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TabGroup/TabGroupExtension.cs

```diff
@@ -10,6 +10,21 @@ public class TabGroupExtension : IMarkdownExtension
     {
         public void Setup(MarkdownPipelineBuilder pipeline)
         {
+            var tabGroupAggregator = new TabGroupAggregator();
+            var aggregateVisitor = new MarkdownDocumentAggregatorVisitor(tabGroupAggregator);
+
+            var tagGroupIdRewriter = new TabGroupIdRewriter();
+            var tagGroupIdVisitor = new MarkdownDocumentVisitor(tagGroupIdRewriter);
+
+            var activeAndVisibleRewriter = new ActiveAndVisibleRewriter();
+            var activeAndVisibleVisitor = new MarkdownDocumentVisitor(activeAndVisibleRewriter);
```

Comment:
>var activeAndVisibleVisitor = new MarkdownDocumentVisitor(activeAndVisibleRewriter); [](start = 12, length = 84)

var activeAndVisibleVisitor = new MarkdownDocumentVisitor(new ActiveAndVisibleRewriter());

Reply:
This is a copy of old code. I do not intend to change existing styles.